src/vs/workbench/browser/parts/statusbar/statusbarItem.ts

```diff
@@ -121,6 +122,16 @@ export class StatusbarEntryItem extends Disposable {
 			} else {
 				this.hover = this._register(setupCustomHover(this.hoverDelegate, this.container, hoverContents));
 			}
+			this.focusListener.clear();
+			this.focusOutListener.clear();
```

Comment:
These aren't needed as setting the value will dispose the old value automatically